### ISSUE TITLE
Możliwość integrowania z serwisem przez API - wyszukiwanie ofert na początek

### ISSUE THREAD

**Issue by raydeal** [Czy-Jest-Eldorado/feature-requests#35]
Cześć,

Czy byłaby możliwość wystawienia API webowego, żeby można było wyszukiwać oferty i otrzymywać odpowiedź w formacie json?

**Comment by jkobus** [comment]
Cześć,

Tak, dodam to do baklogu. Rozumiem, że kolega z forum pythona :)
Czego tam dokładnie potrzebujecie?

**Comment by raydeal** [comment]
Tak, z [python-forum.pl](python-forum.pl) :) Mamy tam kategorie "Praca" i chcielibyśmy dodawać tam oferty jakie się pojawiają na rynku a są związane z Pythonem (stanowisko, wymagania, płaca itd.)  Przydałaby się data dodania, może też aktualizacji, żeby wiedzieć które oferty są nowe/zaktualizowane i możliwość filtrowania po tych datach.
Napisze na maila, żeby dogadać szczegóły.

**Comment by jkobus** [comment]
Dzięki za rozwinięcie. 

Generalnie możemy to brainstormować tutaj. Przygotujcie listę pól, jakie są Wam potrzebne a ja powiem, co jest możliwe. Co do samego sposobu filtrowania, sortowania - możemy dać wszystko to co jest w normalnym wyszukiwaniu. 

Czyli, to co chciałbym ustalić, to:
- lista pól (ewentualnie przykładowy json)
- częstotliwość z jaką chcecie aktualizować dane

**Comment by raydeal** [comment]
Ok, możemy tutaj. Odpowiadając na pytanie "częstotliwość z jaką chcecie aktualizować dane" - to zależy jak często aktualizujesz dane w serwisie, ale myślę, że nie potrzebujemy częściej niż raz na dzień.
Jeśli chodzi o listę pól to poprzeglądam jakie informacje sią dostępne w czyjesteldorado.pl, pomyśle nad tym co by nam się przydało i wrzucę tutaj.

**Comment by raydeal** [comment]
Jako response spodziewalibyśmy się listy z obiektami zawierającymi pola (z tych co widać w ogłoszeniach): "job_title", "company", "salary", "techstack" (to jest to co w tagach pod zarobkami, jako lista), "experience", "employment_type", "contract_type", "work_model", "location", "post_source", "creation_date", "update_date" (jeśli istnieje)

**Comment by raydeal** [comment]
Właściwie to nie trzeba budować API, wystarczy na takiej stronie dodać RSS feed i to powinno załatwić wszystko.

**Comment by Volt-hf** [comment]
@jkobus czy działałeś może w tym temacie? Podpiąłbym się pod FR. 

Do mojego badania chciałbym szukać po konkretnych kategoriach (np. Security) i nazwie stanowiska i uzyskać listę wszystkich aktywnych ogłoszeń z polami takimi jak title, experience i link do docelowego ogłoszenia.
